Sources/ProjectSpec/Linkage.swift

```diff
@@ -42,3 +43,22 @@ extension Target {
         }
     }
 }
+
+extension PBXTarget {
+
+    public var defaultLinkage: Linkage {
```

Comment:
This is the same logic as above. Let's move to an extension on `PBXProductType` in this file